packages/app/src/specs/generators/component/ComponentGeneratorStepOne.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div class="flex flex-col flex-grow justify-between">
+    <template v-if="generatedSpecError">
```

Comment:
I looked to see if I can find anywhere that `generatedSpecError` gets returned but couldn't find any, looks like it is a holdover from https://github.com/cypress-io/cypress/pull/19862/files where if the user generated a component spec that didn't match `specPattern`, it would take them through the `EmptyGenerator` flow so that the user could input the filename manually. Since we don't have this issue with this new flow, can we remove the instances of `erroredCodegenCandidate`?

I know there is another issue where you want to tackle customSpecPatterns (which looks like what this code was trying to solve).